### DISCUSSION TITLE
Deploy fastapi with csv file

### DISCUSSION THREAD

**Discussion by eavolpe** [deta/cloud-docs#484] [Help]
I have managed to create a simple api in using fastapi which listens to a certain column in the provided csv file and if it is found then the api returns some other information stored in the same column. The code is working locally, it is quite simple as it only uses pandas and fastapi, but when I uploaded the project to deta, there seems to be no response from the api. What is the procedure so that that csv file is included in the deta upload?. Thanks a lot for your help.

**Comment by eavolpe** [comment]
Should I upload the code as part of the question? Is this funcionality working/ is the expected behaviour?

**Comment by abdelhai** [comment]
hi. you can include the code that reads the csv. make sure the csv is part of your source code and that it is not stored somewhere else

**Comment by Norrica** [comment]
I have similar issue, with JSON file. 

My app reads its contents to create endpoints, everything is in `main.py`, but when I deploy and run, I get `[Errno 2] No such file or directory: 'myfile.json'`

I tried ./myfile.json , it doesn't help.

All files located in the same folder.

Is Deta even supports additional files?

**Reply by Norrica** [reply]
I fixed this issue.

The problem was that deta does not support cyrillic file names.
